recipe/meta.yaml

```
@@ -87,7 +87,12 @@ requirements:
     - aws-sdk-cpp
 
 test:
+  files:
+    - test_netcdf.py
+  requires:
+    - netcdf4
   commands:
+    - {{ PYTHON }} -c "import test_netcdf; test_netcdf.test_netcdf()"
```

This should be fine but we don't need this with the `.bat` test file.

```suggestion
    - python -c "import test_netcdf; test_netcdf.test_netcdf()"
```

---

@zklaus tried this before (`python` instead of `{{ PYTHON }}`).  But on windows, the test environment is simply never being created and neither the tests in `meta.yaml` nor the one in `run_test.bat` are actually being run.

---

Unfortunately, I can't find the logs for that earlier test -- it seems like they were removed when we started new tests.